javascript/src/main/resources/javascript/SelectOwnerPriority.use.js

```diff
@@ -344,6 +346,33 @@ var SelectOwnerPriority = ( function () {
     }
 
 
+    function getEditionPriorities( manifestations ) {
+        Log.trace( "Entering: ManifestationInfo.getEditionPriorities function" );
+
+        var values = {};
+        for ( var id in manifestations ) {
+            var version = manifestations[id][KEYS].version;
+            var match;
+            var edition;
+            if ( version === null ) {
+                edition = 25;
+            } else if ( ( match = version.match( /^(\d+)\.? udgave/ ) ) ) {
+                var edition = parseInt( match[1] );
+            } else if ( ( match = version.match( /^(\d+)\.? bogklubudgave/ ) ) ) {
```

Comment:
ville det ikke ramme bedre hvis man kun forsøgte at matche på 'bogklub' i stedet for 'bogklubudgave' ?